src/game/boe.main.cpp

```diff
@@ -1296,6 +1295,5 @@ void pause(short length) {
 	// Before pausing, make sure the screen is updated.
 	redraw_screen(REFRESH_NONE);
 	
-	if(get_bool_pref("DrawTerrainFrills", true))
```

Comment:
Removing this should treat it as if the preference is always false. Thus, this becomes "if false then sleep", meaning that the sleep should also be deleted.

Reply:
Oh, I suddenly realized that we want it to always be _true_, not false, so ignore this.